starter-code/routes/celebrities.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
+const router  = express.Router();
+const  celebModel= require('../models/celebrity');
+const { response } = require('../app');
+
+router.get('/celebrities', (req, res) =>{
+    celebModel.find()
+      .then((celebrities)=> res.render('celebrities/index.hbs', {celebrities}))
+      .catch((err)=> console.log ('Error', err))
+  })
+
+router.get('/celebrities/new', (req, res) =>{
+  res.render('celebrities/new.hbs')
+})
+
+router.post('/celebrities/new', (req, res) =>{
+  celebModel.create(req.body)
+    .then(()=>res.redirect('../celebrities'))
+    .catch(()=>res.render('celebrities/new.hbs'))
```

Comment:
for managing .catch in BD requests you have many options, however sending the same page without an error is not recommended. As the user will never know what is happening. One approach would be rendering the same page but with an error message, another would be using the next() function that will basically jump to the next routes if eventually show the 404 page.